packages/vscode/src/extension.ts

```diff
@@ -2,7 +2,7 @@ import { ExtensionContext } from 'vscode'
 import plugins from './plugins'
 
 export function activate(context: ExtensionContext): void {
-  void plugins.map(async (plugin) => {
+  plugins.map(async (plugin) => {
```

Comment:
This should still be tagged void :)
```suggestion
  void plugins.map(async (plugin) => {
```